### PR TITLE
Style for Acta Physica Sinica (物理学报)

### PR DESCRIPTION
**Pull request by redleafnew**
Style for Acta Physica Sinica (物理学报) https://wulixb.iphy.ac.cn/news/tougaoxuzhi.htm

### CODE REVIEW

**Review by adam3smith — CHANGES_REQUESTED** [review]
Some requested changes -- biggest concern is the title-short for language, which is IMO not a good way of handling this.

File: acta-physica-sinica.csl [hunk 1]
```diff
@@ -0,0 +1,445 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="in-text" names-delimiter=". " name-as-sort-order="all" sort-separator=" " demote-non-dropping-particle="never" initialize-with=" " initialize-with-hyphen="false" page-range-format="expanded" default-locale="zh-CN">
+  <info>
+    <title>Acta Physica Sinica (物理学报)</title>
+    <id>http://www.zotero.org/styles/acta-physica-sinica</id>
+    <link href="http://www.zotero.org/styles/acta-physica-sinica" rel="self"/>
+    <link href="https://wulixb.iphy.ac.cn/news/tougaoxuzhi.htm" rel="documentation"/>
```
[the remark this answers is not in the document]
Comment (redleafnew) [reply acta-physica-sinica.csl:7]: Yes, I started this template style, with short title contain the translated version of the item.
And some other styles I created used this methold such as <https://github.com/redleafnew/Chinese-STD-GB-T-7714-related-csl/blob/main/14tcsae.csl>, however, there are two layouts, which is not allowed in cls 1.01 and only support by Jurism.

File: acta-physica-sinica.csl [hunk 2]
```diff
@@ -0,0 +1,445 @@
...
+    <date variable="accessed" form="numeric" prefix="[" suffix="]"/>
+  </macro>
+  <!-- 主要责任者 -->
+  <macro name="author">
+    <names variable="author">
+      <name>
+        <name-part name="family"/>
+        <name-part name="given"/>
+      </name>
+      <substitute>
```
[the remark this answers is not in the document]
Comment (redleafnew) [reply acta-physica-sinica.csl:63]: I have no idea about this either, I have not submit a paper to this journal, only the authors and traslators are needed in the guidelines and some other example articles I downloaded from it's website.

File: acta-physica-sinica.csl [hunk 3]
```diff
@@ -0,0 +1,445 @@
...
+        <text macro="monograph-layout"/>
+      </else>
+    </choose>
+  </macro>
+  <citation collapse="citation-number" after-collapse-delimiter=",">
+    <layout vertical-align="sup" delimiter="," prefix="[" suffix="]">
+      <text macro="citation-layout"/>
+    </layout>
+  </citation>
+  <bibliography entry-spacing="0" et-al-min="10000" et-al-use-first="1000" second-field-align="flush">
```
[the remark this answers is not in the document]
Comment (redleafnew) [reply acta-physica-sinica.csl:439]: ok, I will edit the file.

File: acta-physica-sinica.csl [hunk 4]
```diff
@@ -0,0 +1,445 @@
...
+        <text variable="volume"/>
+      </group>
+      <text variable="issue" prefix="(" suffix=")"/>
+    </group>
+  </macro>
+  <!-- 专著和电子资源 -->
+  <macro name="monograph-layout">
+    <!-- 如果是中文，在short title有内容，添加short title中的内容，即翻译的英文格式 -->
+    <choose>
+      <if variable="title-short">
```
[the remark this answers is not in the document]
Comment (redleafnew) [reply acta-physica-sinica.csl:219]: the original-title field could not be found in Zotero, put this in the Extra?

File: acta-physica-sinica.csl [hunk 5]
```diff
@@ -0,0 +1,445 @@
...
+      <text macro="publishing"/>
+    </group>
+  </macro>
+  <!-- 连续出版物中的析出文献 -->
+  <macro name="article-in-periodical-layout">
+    <!-- 如果是中文，在short title有内容，添加short title中的内容，即翻译的英文格式 -->
+    <choose>
+      <!-- <if language="中文"> -->
+      <if variable="title-short">
+        <text variable="title-short" suffix="(in Chinese) ["/>
```
[the remark this answers is not in the document]
Comment (redleafnew) [reply acta-physica-sinica.csl:354]: Does it like this <code><group delimiter=" " prefix="["=""></code>

File: acta-physica-sinica.csl (hunk 4 above)
[the remark this answers is not in the document]
Comment (redleafnew) [reply acta-physica-sinica.csl:219]: ` original-title` is a CSL variable has no corresponding Zotero field.

File: acta-physica-sinica.csl (hunk 5 above)
[the remark this answers is not in the document]
Comment (redleafnew) [reply acta-physica-sinica.csl]: Using `suffix="]"` in the group will produce a un necessary` ']'` if there is no `original-title` or space if there is `original-title`

[the remark this answers is not in the document]
Comment (redleafnew) [reply acta-physica-sinica.csl]: And using ` <group delimiter=" " prefix="[">` will produce a un necessary '[' if there is no `original-title`, so I think I have to change back to `suffix="(in Chinese) ["/`

**Comment by redleafnew** [comment]
I have changed the file structure with  `<text variable="title-short" suffix=" (in Chinese)" />` and  `<group delimiter=" " prefix=" [" suffix="]">`,  and delete et al min which hope to meet the requirement.

File: acta-physica-sinica.csl (hunk 1 above)
[the remark this answers is not in the document]
Comment (bwiernik) [reply acta-physica-sinica.csl:8]: What he means is please add a link to the style you modified here with `relation="template"`. This is to acknowledge the work of the author of that style and track relationships among styles

[the remark this answers is not in the document]
Comment (redleafnew) [reply acta-physica-sinica.csl:9]: Got it, the link was added.

**Review by redleafnew — COMMENTED** [review]
fixed

**Comment by bwiernik** [comment]
@adam3smith You've already looked at this, so I will let you finish

**Comment by redleafnew** [comment]
thanks.

**Comment by redleafnew** [comment]
Any change requested?

**Comment by redleafnew** [comment]
Any change requested?
@adam3smith

**Comment by adam3smith** [comment]
Sorry for the delay & thanks!

**Pull request merged**